### PR TITLE
Make the post editor first responder on blank new posts

### LINKED ISSUE #115: Auto-focus on Title field upon loading empty editor
This is a tracking issue to bring the Mac app to feature parity with the iOS app by implementing #104 for Mac.

That means:

- [x] Title field should become first responder on load
- ~~Return key should navigate from title field to body field~~ (dropped in favour of single-field editing on Mac)
- ~~The title field should expand its height as the title wraps beyond one line~~ (to do in #133)

### PR DESCRIPTION
**Pull request by AngeloStavrow**
Closes #115.

### CODE REVIEW

File: macOS/PostEditor/PostTextEditingView.swift [hunk 1]
```diff
@@ -2,65 +2,98 @@ import SwiftUI
...
-                        .font(.custom(appearance.rawValue, size: 17, relativeTo: .body))
-                }
-                TextEditor(text: $post.body)
+        ZStack(alignment: .topLeading) {
+            if combinedText.count == 0 {
+                Text("Write…")
+                    .foregroundColor(Color(NSColor.placeholderTextColor))
+                    .padding(.horizontal, 5)
                     .font(.custom(appearance.rawValue, size: 17, relativeTo: .body))
-                    .lineSpacing(17 * bodyLineSpacingMultiplier)
```
Comment (thebaer) [inline macOS/PostEditor/PostTextEditingView.swift:38]: I'm unable to test, so I'm not sure -- but does this mean the Mac app won't have the same line spacing as the iOS app?

Comment (AngeloStavrow) [reply macOS/PostEditor/PostTextEditingView.swift:38]: Ah yeah, forgot to add that back! Let me fix that.

Comment (AngeloStavrow) [reply macOS/PostEditor/PostTextEditingView.swift:38]: Before 2e56d62:

Screen Shot 2020-11-24 at 14 31 53

After 2e56d62:

Screen Shot 2020-11-24 at 14 34 09

Comment (thebaer) [reply macOS/PostEditor/PostTextEditingView.swift:38]: Looks great!

**Review by thebaer — APPROVED** [review]
Looks good to me after only reading through the code :+1:

**Pull request merged**